doc/source/cd/background.rst

```diff
@@ -0,0 +1,843 @@
+Background
+==========
+
+.. contents:: :local:
+   :depth: 2
+
+0. What is drift?
```

Comment:
Do we want to start numbering from 0. instead of 1.?

Reply:
What is your preference? I was copying `examples/alibi_detect_deploy.html` which starts from 0. but happy to change if 1. if you prefer?

Reply:
I think you need to blame @arnaudvl for that! He has the final word...

Reply:
I am impartial, whatever reads the easiest.

Reply:
Although starting from 0. is nicely pythonic, I feel like 1.0 might be a bit more readable, so have gone with that. Updated `examples/alibi_detect_deploy.html` too.